src/pages/blog/index.tsx

```diff
@@ -122,25 +122,32 @@ const BlogPage: FC<{ data: BlogGQL }> = ({ data }) => {
   const { componentType, isDesktop } = useComponentType();
   const developerProfile = useDeveloperProfile();
   const [selectedBlogpost, setSelectedBlogpost] = useState(-1);
+  const dateFormat = 'DD MMMM YYYY';
+
+  if (typeof window !== `undefined`) {
```

Comment:
Why nested if?

Reply:
No idea, I was fixing it chaotically after the production build failed,  changed to one `if` 👌 
Fun fact, Gatsby production build is SSR so we need to check for `window` if we want to reference it, for example, the `navigate` function under the hood uses it and it was breaking the build.